index.html

```diff
@@ -76,132 +82,117 @@ <h3 class="H-title">I'am a Software Developer</h3>
                 </div>
               </section>
 
-              <!--Projects section-->
-              <section id="projects">
-                <div id="h2-container">
-                  <h2 class="h2-project">My Recent Works</h2>
-                  <div id="h2-div"></div>
-                </div>
-                <div id="portfolio-grid">
-                  <div id="main-project">
-                    <div id="main-image-div">
-                      <img
-                        src="./images/Img.png"
-                        alt="project-works"
-                        id="project-works"
-                        data-img="image"
-                      />
-                    </div>
-                    <div id="project-content">
-                      <h3 id="project-h3" data-title="title">Multi-Post Stories</h3>
-                      <p id="content-p">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required. has been the industry's standard dummy text
-                        ever since the 1500s, when an unknown printer took a standard
-                        dummy text. A daily selection of privately personalized reads; no
-                        accounts or sign-ups required.
-                      </p>
-                      <div>
-                      <ul class="language-button12">
-                        <li>html</li>
-                        <li>css</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                      </ul>
-                      <button
-                        type="button"
-                        id="main-project-button"
-                        class="btn project-btn"
-                        data-btn="#main-project">
-                        See Project
-                      </button>
-                    </div>
-                    </div>
-                  </div>
-                </div>
-              </section>
+<section class="proje1">
+  <div class="roject__recent">
+    <h2 class="recent__h2">My Recent Works</h2><hr class="recent__h2__line">
+  </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161399883-a2e52d48-b8b1-49d6-a74a-46c509852a5d.png) | ![Screenshot_4](https://user-images.githubusercontent.com/69638013/161399879-086a582a-a2a2-4295-9444-96c45c83f623.png)


- [ ] Kindly align the My recent works title to the left and the line should take the remaining space 👍🏿

Reply:
Good job implementing the changes